src/Illuminate/Contracts/Auth/Guard.php

```diff
@@ -21,7 +21,7 @@ public function guest();
     /**
      * Get the currently authenticated user.
      *
-     * @return \Illuminate\Contracts\Auth\Authenticatable|null
+     * @return \Illuminate\Contracts\Auth\Authenticatable|\App\User|null
```

Comment:
Yes, you have a valid point. 
I just thought it would be convenient for most users.